sootup.java.core/src/main/java/sootup/java/core/JavaSootClass.java

```diff
@@ -627,7 +649,7 @@ public JavaSootClass build() {
         assert instance.classSource != null;
         return new JavaSootClass(
             instance.classSource,
-            instance.sourceType,
+            null,
```

Comment:
why?

Reply:
because there is a null check above. That was a warning that sourceType will be always null. but i guess this is also illegal 😆 i just resolved the warning